src/auth.c

```diff
@@ -1716,13 +1716,16 @@ int pr_auth_chroot(const char *path) {
 
   tz = pr_env_get(session.pool, "TZ"); 
   if (tz == NULL) {
+    char tzbuf[32];
+    long off = (timezone / 60 / 60) - daylight;
+    snprintf(tzbuf, sizeof(tzbuf), "%s%ld", daylight ? tzname[1] : tzname[0], off);
```

Comment:
Huh... that does appear to be working for me now.  I tried it a year or so ago, and it didn't work reliably, presumably because the chroot directory does not have a /etc/localtime file.  But I am running a newer glibc version now, so maybe something has changed on that side?